JoinColony/colonyCDapp
# Fix: No colonies joined UI adjustments

## Description

![no colonies joined](https://github.com/user-attachments/assets/5697cc77-3322-4051-9e5b-8500e3f3f0bf)

## Testing

1. Log in as Dev Wallet 4
2. Verify that the No colonies joined UI matches figma for both web & mobile

Resolves #3090
